pkg/providers/instance/instance.go

```diff
@@ -450,7 +450,8 @@ func filterExoticInstanceTypes(instanceTypes []*cloudprovider.InstanceType) []*c
 		if !resources.IsZero(it.Capacity[v1beta1.ResourceAWSNeuron]) ||
```

Comment:
Considering how long this list is, I wonder if the `v1beta1` labels package should just have this list in there and then we don't continually have to remember to add to this list